tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'll need to check some of this options, did you come up with this file yourself or was it generated? some of this configs might be useful for the BE repo too!

Reply:
Auto generated